configure.ac

```diff
@@ -174,7 +174,20 @@ fi
 #~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 new_section "Coq"
 
-AC_CHECK_PROG(COQC,coqc,coqc,no)
+#AC_ARG_VAR(COQBIN, [path to Coq executables [empty]])
+#if test ${COQBIN##*/}; then COQBIN=$COQBIN/; fi
+#AC_MSG_RESULT($COQBIN)
+
+if test "$COQBIN"; then
```

Comment:
Will this work as expected when users set COQBIN to the empty string (which I assume should have the same meaning as having COQBIN not set).

I believe something like `test   "a$COQBIN" != "a"`  is more resilient.